### PR TITLE
Add "about" section to resident status screen

### PR DESCRIPTION
**Pull request by Fruitloopins**
#### Description: 

This adds a /res equivalent of the "board" on /t and /n, I have chosen the word "about" in place of board as I think it fits better for a person. I tried to replicate the functionality of the other board commands as best as I can.

image

____
#### New Nodes/Commands/ConfigOptions: 

- `/res set about [clear/none/reset/msg]`
  - `towny.command.resident.set.about`
- `/ta set about [username] [clear/none/reset/msg]`
  - `towny.command.townyadmin.set.about`

____
#### Relevant Towny Issue ticket:

#6617 

____
- [x] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

File: Towny/src/main/java/com/palmergames/bukkit/towny/TownyFormatter.java [hunk 1]
```diff
@@ -144,6 +144,9 @@ public static StatusScreen getStatus(Resident resident, CommandSender sender) {
 		// First used if last online is this year, 2nd used if last online is early than this year.
 		// Registered: Sept 3 2009 | Last Online: March 7 @ 14:30
 		// Registered: Sept 3 2009 | Last Online: March 7 2009
+		if (resident.getAbout() != null && !resident.getAbout().isEmpty())
+			screen.addComponentOf("about", colourKeyValue(translator.of("status_about"), resident.getAbout()));
+
```
Comment (LlmDl) [inline Towny/src/main/java/com/palmergames/bukkit/towny/TownyFormatter.java:149]: This should be moved up above the comments showing what the registered section looks like. It should also have a brief comment line of its own.

File: Towny/src/main/java/com/palmergames/bukkit/towny/db/SQLSchema.java [hunk 2]
```diff
@@ -179,6 +179,7 @@ private static List<String> getResidentColumns(){
 		columns.add("`friends` mediumtext");
 		columns.add("`metadata` text DEFAULT NULL");
 		columns.add("`uuid` VARCHAR(36) NOT NULL");
+		columns.add("`residentAbout` mediumtext DEFAULT NULL");
```
Comment (LlmDl) [inline Towny/src/main/java/com/palmergames/bukkit/towny/db/SQLSchema.java:182]:
```suggestion
		columns.add("`about` mediumtext DEFAULT NULL");

```

**Review by LlmDl — CHANGES_REQUESTED** [review]

File: Towny/src/main/java/com/palmergames/bukkit/towny/TownyFormatter.java (hunk 1 above)
Comment (Fruitloopins) [reply Towny/src/main/java/com/palmergames/bukkit/towny/TownyFormatter.java:149]: my bad

File: Towny/src/main/java/com/palmergames/bukkit/towny/db/TownySQLSource.java [hunk 3]
```diff
@@ -784,6 +784,12 @@ private boolean loadResident(Resident resident, ResultSet rs) {
 				plugin.getLogger().log(Level.WARNING, "Could not get lastOnline column on the residents table", e);
 			}
 			
+			try {
+				resident.setAbout(rs.getString("about"));
+			} catch (SQLException e) {
+				plugin.getLogger().log(Level.WARNING, "Could not get about column on the residents table");
+			}
+
```
Comment (LlmDl) [inline Towny/src/main/java/com/palmergames/bukkit/towny/db/TownySQLSource.java:792]:
```suggestion
			try {
				line = rs.getString("about");
				if (line != null)
					resident.setAbout(line);
			} catch (SQLException e) {
				plugin.getLogger().log(Level.WARNING, "Could not get about column on the residents table", e);
			}

```

This code will also have to get moved down below the line where `String line;` exists (line 835 on master branch.)

**Review by LlmDl — CHANGES_REQUESTED** [review]
The only other thing I can think about is that the new `towny.command.resident.set.about` node ought to be put into the default townyperms.yml file's nomad section, and a ConfigMigration should be created to put that node into the townyperms.ymls that already exist. I can point you in the right direction on the discord if you need help with that.

**Review by LlmDl — APPROVED** [review]

**Pull request merged**